rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/explorer/UnityExplorerNode.kt

```diff
@@ -73,7 +75,8 @@ class UnityExplorerRootNode(project: Project, private val packageManager: Packag
 open class UnityExplorerNode(project: Project,
                              virtualFile: VirtualFile,
                              nestedFiles: List<VirtualFile>,
-                             private val isUnderAssets: Boolean)
+                             private val isUnderAssets: Boolean,
+                             private val isReadOnlyPackageFile: Boolean = false)
```

Comment:
Because `PackageNode` only represents the node at the root of the package. All of its children are instances of `UnityExplorerNode`, and will get the "ignore" colour, unless it's overridden. Instead of deriving another new class, I added the parameter.

Reply:
Oook